src/ed_object_models/spawn_sdf_in_gazebo.py

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/python
+
+import rospy
+import yaml
+import os
+import glob
+
+from rospkg import RosPack
+from gazebo_msgs.srv import SpawnModel
+from geometry_msgs.msg import Pose
+
+
+def from_yaml(yaml_file):
+    '''
+    Spawns a list of sdf files from a yaml file into Gazebo.
+    :param yaml_file: path to a yaml file.
+    :type yaml_file: str
+    :return: Spawns the sdf models.
+    '''
+
+    # Initialize ROS node
+    rospy.init_node('gazebo_simulator_object_spawner')
+
+    # Wait until gazebo is ready to spawn sdf models
+    rospy.wait_for_service('gazebo/spawn_sdf_model')
```

Comment:
Should this be a relative or absolute topic? Right now it is a relative topic.

Reply:
It works so I assume for now that it can stay this way.